FreshRSS/FreshRSS
# Improved: Enclosures: HTML from Entry.php to enclosures.phtml

Next step after #4944 this PR cuts out the HTML from `Entry.php` and put it into `enclosure.phtml`.

(Ref #4943 where I tried it first time)

Important: This PR creates not the best `.phtml`, because of many `echo` for printing the HTML.

I would prefer to have a PR review here, while it is clear how I cut out the HTML from `Entry.php` and did not change the HTML structure. Improvement PR will follow.

General goal: I want to improve the displaying of enclosures. One step is to bring the output into the `.phtml`. Later I will improve this `.phtml`

Changes proposed in this pull request:

- HTML from `Entry.php` to `enclosure.phtml` as pure extract.


How to test the feature manually:

1. open the normal view or reader view
2. take a look onto articles with enclosures (I support some feeds with enclosures in my repo: https://github.com/math-GH/testfeed)
3. the enclosures will be displayed (and nothing should changed in comparison to before)

Pull request checklist:
